.github/workflows/validate.yml

```diff
@@ -1,27 +1,56 @@
 name: validate-json
 
 on:
-  [push]
-  # push:
-  #   branches: ["main"]
-  # pull_request:
-  #   branches: ["main"]
+  workflow_call:
+    inputs:
+      should_download:
+        description: "Download JSON artifacts if called by another workflow?"
+        type: boolean
+    outputs:
+      is_valid:
+        description: "Is the JSON valid?"
+        value: ${{ jobs.validate.outputs.is_valid}}
+
+  pull_request:
+    branches: ["*"]
+
+  workflow_dispatch:
 
 jobs:
-  build:
+  validate:
     runs-on: ubuntu-latest
+    outputs:
+      is_valid: ${{ steps.save_state.outputs.is_valid}}
+
     steps:
       - uses: actions/checkout@v4
+
+      - name: Download JSON artifacts # if called by another workflow
+        if: ${{ inputs.should_download }}
+        uses: actions/download-artifact@v4
+        with:
+          name: json-files
+          path: json
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          # cache: "pip"
```

Comment:
You can re-enable this, and install deps
```
      - name: Install dependencies
        run: |
          python -m pip install --upgrade pip
          pip install -r requirements.txt
```
Technically this is a separate system, and you're running python so you should probably have the deps installed!

Reply:
Thanks for this!